Add `Community Mentorship Program Summer 2025 Signups Now Open` news post (#13352)
ppy/osu-wiki: news/2025/2025-06-09-community-mentorship-program-summer-2025-signups-now-open.md

Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as speedmapping contests, organise guest difficulties for your mapsets on our job board, and more! 

On the public side of things, [our Discord server](https://discord.com/invite/Ft2FtXmBgx) offers general mapping discussion, feedback channels, the guest difficulty job board and more! We also announce mapping opportunities for tournaments and other relevant events on the server. If you happen to be running an event that you feel is relevant, feel free to reach out to the [Organisation Team](https://docs.google.com/spreadsheets/d/1dVbDmk2Z6RsosS8rN11cfjCPMLKMuCjrICOVqTW2KH4/edit?usp=sharing)!

If you're interested in the program, please read the [forum opening post](https://osu.ppy.sh/community/forums/topics/2079752?n=1) for a full rundown on how to apply. 
The mentee-signups are currently open, and close on the 16th of June! 

### osu! 

> | Mentor      | Timezone     |

Reviewer: ```suggestion
| Mentor | Timezone |
```

| :-- | :-- |

> | ::{ flag=US }:: [IllusionB](https://osu.ppy.sh/users/13292488)   | UTC-8 |

Reviewer: ```suggestion
| ::{ flag=US }:: [IllusionB](https://osu.ppy.sh/users/13292488) | UTC-8 |
```

> | ::{ flag=US }:: [Wanpachi](https://osu.ppy.sh/users/12157130)    | UTC-7 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Wanpachi](https://osu.ppy.sh/users/12157130) | UTC-7 |
```

> | ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410)     | UTC-7 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | UTC-7 |
```

> | ::{ flag=US }:: [Scorf](https://osu.ppy.sh/users/17038784)       | UTC-7 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Scorf](https://osu.ppy.sh/users/17038784) | UTC-7 |
```

> | ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117)       | UTC-5 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117) | UTC-5 |
```

> | ::{ flag=US }:: [Local Hero](https://osu.ppy.sh/users/16134122)  | UTC-5 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Local Hero](https://osu.ppy.sh/users/16134122) | UTC-5 |
```

> | ::{ flag=US }:: [Jennifer](https://osu.ppy.sh/users/6761853)    | UTC-5 |

Reviewer: ```suggestion
| ::{ flag=US }:: [Jennifer](https://osu.ppy.sh/users/6761853) | UTC-5 |
```

> | ::{ flag=CA }:: [Sparhten](https://osu.ppy.sh/users/7601720)    | UTC-3 |

Reviewer: ```suggestion
| ::{ flag=CA }:: [Sparhten](https://osu.ppy.sh/users/7601720) | UTC-3 |
```

> | ::{ flag=DE }:: [Okoayu](https://osu.ppy.sh/users/1623405)      | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=DE }:: [Okoayu](https://osu.ppy.sh/users/1623405) | UTC+2 |
```

> | ::{ flag=DE }:: [\-eNVy-](https://osu.ppy.sh/users/10632422)     | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=DE }:: [\-eNVy-](https://osu.ppy.sh/users/10632422) | UTC+2 |
```

> | ::{ flag=PL }:: [App](https://osu.ppy.sh/users/13650508)         | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=PL }:: [App](https://osu.ppy.sh/users/13650508) | UTC+2 |
```

> | ::{ flag=DE }:: [Kojio](https://osu.ppy.sh/users/2054596)       | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=DE }:: [Kojio](https://osu.ppy.sh/users/2054596) | UTC+2 |
```

> | ::{ flag=ES }:: [flouah](https://osu.ppy.sh/users/8030129)      | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=ES }:: [flouah](https://osu.ppy.sh/users/8030129) | UTC+2 |
```

> | ::{ flag=FR }:: [Kyuukai](https://osu.ppy.sh/users/5337374)     | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=FR }:: [Kyuukai](https://osu.ppy.sh/users/5337374) | UTC+2 |
```

> | ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882)        | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | UTC+2 |
```

> | ::{ flag=IT }:: [zekk](https://osu.ppy.sh/users/9704802)        | UTC+2 |

Reviewer: ```suggestion
| ::{ flag=IT }:: [zekk](https://osu.ppy.sh/users/9704802) | UTC+2 |
```

| ::{ flag=RU }:: [NeKroMan4ik](https://osu.ppy.sh/users/11387664) | UTC+3 |

> | ::{ flag=FI }:: [Nyanaro](https://osu.ppy.sh/users/4157611)     | UTC+3 |

Reviewer: ```suggestion
| ::{ flag=FI }:: [Nyanaro](https://osu.ppy.sh/users/4157611) | UTC+3 |
```

> | ::{ flag=KZ }:: [\-Hitomi](https://osu.ppy.sh/users/15583987)    | UTC+5 |

Reviewer: ```suggestion
| ::{ flag=KZ }:: [\-Hitomi](https://osu.ppy.sh/users/15583987) | UTC+5 |
```

> | ::{ flag=JP }:: [MochiA](https://osu.ppy.sh/users/9312734)      | UTC+9 |

Reviewer: ```suggestion
| ::{ flag=JP }:: [MochiA](https://osu.ppy.sh/users/9312734) | UTC+9 |
```

You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1PrL5cXwU-EMIkUaRKPHbRsi-AUeicQhZZR2QkjVDevM/edit) and apply to them [here](https://forms.gle/j9tUAjMf47JoRCRK8). 

### osu!mania 

> | Mentor          | Timezone |

Reviewer: ```suggestion
| Mentor | Timezone |
```

| :-- | :-- |

> | ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773)          | UTC-4    |

Reviewer: ```suggestion
| ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773) | UTC-4 |
```

> | ::{ flag=NL }:: [Muse Dash](https://osu.ppy.sh/users/13695676)       | UTC+2    |

Reviewer: ```suggestion
| ::{ flag=NL }:: [Muse Dash](https://osu.ppy.sh/users/13695676) | UTC+2 |
```

> | ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640)   | UTC+3    |

Reviewer: ```suggestion
| ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640) | UTC+3 |
```

> | ::{ flag=VN }:: [AHHHHHHHHHHHHHH](https://osu.ppy.sh/users/8249608) | UTC+7    |

Reviewer: ```suggestion
| ::{ flag=VN }:: [AHHHHHHHHHHHHHH](https://osu.ppy.sh/users/8249608) | UTC+7 |
```

> | ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785)           | UTC+7    |

Reviewer: ```suggestion
| ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) | UTC+7 |
```

| ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534)         | UTC+8    |

> | ::{ flag=KR }:: [Dreamwalker](https://osu.ppy.sh/users/8946550)     | UTC+9    |

Reviewer: ```suggestion
| ::{ flag=KR }:: [Dreamwalker](https://osu.ppy.sh/users/8946550) | UTC+9 |
```

---
layout: post
title: Community Mentorship Program Summer 2025 Signups Now Open
date: 2025-06-09 13:00:00 +0000
---

The 30th cycle of the Community Mentorship Program is upon us! Get ready to delve deep into the world of mapping, modding, or storyboarding.

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

The Community Mentorship Program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.

Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as speedmapping contests, organise guest difficulties for your mapsets on our job board, and more! 

> On the public side of things, [our Discord server](https://discord.com/invite/Ft2FtXmBgx) offers general mapping discussion, feedback channels, the guest difficulty job board and more! We also announce mapping opportunities for tournaments and other relevant events on the server. If you happen to be running an event that you feel is relevant, feel free to reach out to the [Organisation Team](https://docs.google.com/spreadsheets/d/1dVbDmk2Z6RsosS8rN11cfjCPMLKMuCjrICOVqTW2KH4/edit?usp=sharing)!

Reviewer 2: "organisation team" is a generic term so it feels weird to capitalise

```suggestion
On the public side of things, [our Discord server](https://discord.com/invite/Ft2FtXmBgx) offers general mapping discussion, feedback channels, the guest difficulty job board and more! We also announce mapping opportunities for tournaments and other relevant events on the server. If you happen to be running an event that you feel is relevant, feel free to reach out to the [organisation team](https://docs.google.com/spreadsheets/d/1dVbDmk2Z6RsosS8rN11cfjCPMLKMuCjrICOVqTW2KH4/edit?usp=sharing)!
```

> If you're interested in the program, please read the [forum opening post](https://osu.ppy.sh/community/forums/topics/2079752?n=1) for a full rundown on how to apply. 

Reviewer 2: ```suggestion
If you're interested in the program, please read the [forum post](https://osu.ppy.sh/community/forums/topics/2079752?n=1) for a full rundown on how to apply. 
```

> The mentee-signups are currently open, and close on the 16th of June! 

Reviewer 2: ```suggestion

The mentee signups are currently open, and close on the 16th of June! 
```

### osu! 

> | Mentor | Timezone |

Reviewer 2: https://en.wikipedia.org/wiki/Time_zone

```suggestion
| Mentor | Time zone |
```

| ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117) | UTC-5 |
| ::{ flag=US }:: [Local Hero](https://osu.ppy.sh/users/16134122) | UTC-5 |
| ::{ flag=US }:: [Jennifer](https://osu.ppy.sh/users/6761853) | UTC-5 |
| ::{ flag=CA }:: [Sparhten](https://osu.ppy.sh/users/7601720) | UTC-3 |
| ::{ flag=DE }:: [Okoayu](https://osu.ppy.sh/users/1623405) | UTC+2 |
| ::{ flag=DE }:: [\-eNVy-](https://osu.ppy.sh/users/10632422) | UTC+2 |
| ::{ flag=PL }:: [App](https://osu.ppy.sh/users/13650508) | UTC+2 |
| ::{ flag=DE }:: [Kojio](https://osu.ppy.sh/users/2054596) | UTC+2 |
| ::{ flag=ES }:: [flouah](https://osu.ppy.sh/users/8030129) | UTC+2 |
| ::{ flag=FR }:: [Kyuukai](https://osu.ppy.sh/users/5337374) | UTC+2 |
| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | UTC+2 |
| ::{ flag=IT }:: [zekk](https://osu.ppy.sh/users/9704802) | UTC+2 |
| ::{ flag=RU }:: [NeKroMan4ik](https://osu.ppy.sh/users/11387664) | UTC+3 |
| ::{ flag=FI }:: [Nyanaro](https://osu.ppy.sh/users/4157611) | UTC+3 |
| ::{ flag=KZ }:: [\-Hitomi](https://osu.ppy.sh/users/15583987) | UTC+5 |
| ::{ flag=JP }:: [MochiA](https://osu.ppy.sh/users/9312734) | UTC+9 |

> You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1PrL5cXwU-EMIkUaRKPHbRsi-AUeicQhZZR2QkjVDevM/edit) and apply to them [here](https://forms.gle/j9tUAjMf47JoRCRK8). 

Reviewer 2: no need for url shortener, only causes unnecessary redirect

```suggestion
You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1PrL5cXwU-EMIkUaRKPHbRsi-AUeicQhZZR2QkjVDevM/edit) and apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSfs10qNps9y4DrTcUjZEc5u28bpqUdHXaxDrcUDWGhmufN9eA/viewform). 
```

### osu!mania 

| Mentor | Timezone |
| :-- | :-- |
| ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773) | UTC-4 |
| ::{ flag=NL }:: [Muse Dash](https://osu.ppy.sh/users/13695676) | UTC+2 |
| ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640) | UTC+3 |
| ::{ flag=VN }:: [AHHHHHHHHHHHHHH](https://osu.ppy.sh/users/8249608) | UTC+7 |
| ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) | UTC+7 |
| ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534) | UTC+8 |
| ::{ flag=KR }:: [Dreamwalker](https://osu.ppy.sh/users/8946550) | UTC+9 |

> You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1g2KhhmBqiqNAe2I2PJPicaSFtWPQ5sScRluC9oQzFGg/edit) and apply to them [here](https://forms.gle/5pwB9qqUsDc3kV6H9).

Reviewer 2: ```suggestion
You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1g2KhhmBqiqNAe2I2PJPicaSFtWPQ5sScRluC9oQzFGg/edit) and apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSeVdfghHjpjOiGJ1rw167rOsuAczWgub0sYYX_wYCP1_kwrTw/viewform).
```

### osu!taiko

> | Mentor | Timezone |

Reviewer 2: ```suggestion
| Mentor | Time zone |
```

| :-- | :-- |
| ::{ flag=DE }:: [Broadsword](https://osu.ppy.sh/users/11357694) | UTC+2 |
| ::{ flag=FR }:: [Kin](https://osu.ppy.sh/users/480689) | UTC+2 |

> You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1_bMj04v4kUhdBd_JS6yiex9pBSFcxh_Uunn--n7Ugt8/edit) and apply to them [here](https://forms.gle/unAie67mgosQK6vE7).

Reviewer 2: ```suggestion
You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/1_bMj04v4kUhdBd_JS6yiex9pBSFcxh_Uunn--n7Ugt8/edit) and apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSfb7Am7-lX0IzFNCCpY7gsACNzv9gcc_5KAEAK5wYVNiNFk4w/viewform).
```

### osu!catch

> | Mentor | Timezone |

Reviewer 2: ```suggestion
| Mentor | Time zone |
```

| :-- | :-- |
| ::{ flag=US }:: [Usaha](https://osu.ppy.sh/users/6443117) | UTC-5 |
| ::{ flag=DE }:: [Du5t](https://osu.ppy.sh/users/6053071) | UTC+2 |
| ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776) | UTC+2 |
| ::{ flag=PL }:: [Verti](https://osu.ppy.sh/users/10674528) | UTC+2 |

> You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/16J1nxWgHadycR2Qu232VufdcmHKPXWb6MFPixO-ehlg/edit) and apply to them [here](https://forms.gle/YrbXSstDdvBGGy2m7).

Reviewer 2: ```suggestion
You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/16J1nxWgHadycR2Qu232VufdcmHKPXWb6MFPixO-ehlg/edit) and apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSdoi7VNJNfEv6V3aJ953255smnpBBqRX5XzUcOmACWeq3bG6Q/viewform).
```

### Modding

> | Mentor | Game mode | Timezone |

Reviewer 2: ```suggestion
| Mentor | Game mode | Time zone |
```

| :-- | :-- | :-- |
| ::{ flag=KR }:: [KeyWee](https://osu.ppy.sh/users/10476879) | osu! | UTC-8 |
| ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | osu! | UTC-7 |
| ::{ flag=CL }:: [melleganol](https://osu.ppy.sh/users/16452371) | osu! | UTC-4 |
| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | osu! | UTC+2 |
| ::{ flag=RU }:: [Daycore](https://osu.ppy.sh/users/5596337) | osu! | UTC+3 |
| ::{ flag=TH }:: [Electoz](https://osu.ppy.sh/users/6485263) | osu! | UTC+7 |
| ::{ flag=KR }:: [Spectator](https://osu.ppy.sh/users/702598) | osu!catch | UTC+9 |
| ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773) | osu!mania | UTC-4 |
| ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534) | osu!mania | UTC+8 |
| ::{ flag=KR }:: [secXcscX](https://osu.ppy.sh/users/13543418) | osu!mania | UTC+9 |
| ::{ flag=FR }:: [Kin](https://osu.ppy.sh/users/480689) | osu!taiko | UTC+2 |
| ::{ flag=RU }:: [\_HeLLFly\_](https://osu.ppy.sh/users/14225226) | osu!taiko | UTC+3 |

For modding, each game mode has their separate overview: 

> - [osu! Mentors](https://docs.google.com/spreadsheets/d/1hXMTAFDdhjpQP72WyfDkjOnZECf6DO_Ba5sxjpowUKw/edit?gid=2064960339#gid=2064960339)
> - [osu!mania Mentors](https://docs.google.com/spreadsheets/d/1AcZHY7OylKGEgzgfryqLr9FOI6-oXdNqPT2Zistz3Yo/edit?gid=2064960339#gid=2064960339)
> - [osu!taiko Mentors](https://docs.google.com/spreadsheets/d/1cjhM_wRE-BmH5v1iVF_uP1Jp6uLNamS7aCnjAQmEQb4/edit?gid=2064960339#gid=2064960339)
> - [osu!catch Mentors](https://docs.google.com/spreadsheets/d/1Ac7_geRabvpRb60dxeLWYhIYzj9D_aDNzw2P8ZlhJ08/edit?gid=2064960339#gid=2064960339)

Reviewer 2: ```suggestion
- [osu! mentors](https://docs.google.com/spreadsheets/d/1hXMTAFDdhjpQP72WyfDkjOnZECf6DO_Ba5sxjpowUKw/edit?gid=2064960339#gid=2064960339)
- [osu!mania mentors](https://docs.google.com/spreadsheets/d/1AcZHY7OylKGEgzgfryqLr9FOI6-oXdNqPT2Zistz3Yo/edit?gid=2064960339#gid=2064960339)
- [osu!taiko mentors](https://docs.google.com/spreadsheets/d/1cjhM_wRE-BmH5v1iVF_uP1Jp6uLNamS7aCnjAQmEQb4/edit?gid=2064960339#gid=2064960339)
- [osu!catch mentors](https://docs.google.com/spreadsheets/d/1Ac7_geRabvpRb60dxeLWYhIYzj9D_aDNzw2P8ZlhJ08/edit?gid=2064960339#gid=2064960339)
```

You can apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSedojsQxwaUWRPZ_AeP_bMr3GNsWMy62MpVwHQfyPtQfeSxXw/viewform).

### Storyboarding

| Mentor | Timezone |
| :-- | :-- |
| ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323) | UTC-4 |
| ::{ flag=HU }:: [Himada](https://osu.ppy.sh/users/10959366) | UTC+2 |
| ::{ flag=IT }:: [Kuroise](https://osu.ppy.sh/users/14547127) | UTC+2 |
| ::{ flag=MY }:: [R3aCt10n](https://osu.ppy.sh/users/17020051) | UTC+8 |

> You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/18vLaEyRnsfc2hpUb53EpggRwY6Vg7PqTScbs2qI6otc/edit) and apply to them [here](https://forms.gle/5bh7XsZw4E5yX4QNA).

Reviewer 2: ```suggestion
You can find more information on these mentors in their [detailed overview](https://docs.google.com/spreadsheets/d/18vLaEyRnsfc2hpUb53EpggRwY6Vg7PqTScbs2qI6otc/edit) and apply to them [here](https://docs.google.com/forms/d/e/1FAIpQLSewCsK_L1pTr2vAMYiLtQZIfRMjqYLuepsBJNwEIYFDBM0P1A/viewform).
```

---

> If you have further questions, don't hesitate to reach out to the [Organisation Team](https://docs.google.com/spreadsheets/d/1dVbDmk2Z6RsosS8rN11cfjCPMLKMuCjrICOVqTW2KH4/edit?usp=sharing) or [join our Discord server!](https://discord.com/invite/Ft2FtXmBgx)

Reviewer 2: ```suggestion
If you have further questions, don't hesitate to reach out to the [organisation team](https://docs.google.com/spreadsheets/d/1dVbDmk2Z6RsosS8rN11cfjCPMLKMuCjrICOVqTW2KH4/edit?usp=sharing) or [join our Discord server](https://discord.com/invite/Ft2FtXmBgx)!
```
